lms/static/js/bookmarks/views/bookmarks_list_button.js

```diff
@@ -16,11 +16,20 @@
             },
 
             initialize: function () {
-                this.bookmarksListView = new BookmarksListView({
-                    collection: new BookmarksCollection(),
-                    loadingMessageView: new MessageView({el: $(this.loadingMessageElement)}),
-                    errorMessageView: new MessageView({el: $(this.errorMessageElement)})
-                });
+                var bookmarksCollection = new BookmarksCollection(
+                    {
+                        course_id: $('.courseware-results').data('courseId'),
+                        url: $(".courseware-bookmarks-button").data('bookmarksApiUrl')
```

Comment:
I'm still wondering why these parameters are embedded in the DOM. Are they only used by the bookmarks JS? If so, I'd like to see them passed to the collection using the method we usually use: in your `main.js` or factory JS file, pass through all the variables you need from the server.